packages/ui-components/src/Icon/Icon.module.scss

```diff
@@ -30,7 +33,7 @@
 }
 
 .size-default {
-  height: 24px;
+  height: 46px;
```

Comment:
This height isn't specified in the design at all. This feels like a specific number for use in a context (like an Input) and doesn't belong here.

Reply:
yup, updated these values from figma so they reflect latest design icon sizes.